packages/report-server/src/reportBuilder/transform/parser/functions/context.ts

```diff
@@ -48,9 +48,27 @@ export const orgUnitIdToCode: ContextFunctionConfig = {
   func: ({ getContext }) => (orgUnitId: string) => {
     const { orgUnits } = getContext();
     if (!orgUnits) {
-      throw new Error("Missing dependency 'orgUnits' required by 'orgUnitIdToCode'");
+      throw new Error("Missing dependency 'orgUnits' required by 'orgUnitIdToCode()'");
     }
     const idToCode = reduceToDictionary(orgUnits, 'id', 'code');
     return idToCode[orgUnitId];
   },
 };
+
+export const orgUnitAttribute: ContextFunctionConfig = {
+  dependencies: ['orgUnits'],
+  func: ({ getContext }) => (orgUnitCode: string, attributeKey: string) => {
+    const { orgUnits } = getContext();
+    if (!orgUnits) {
+      throw new Error("Missing dependency 'orgUnits' required by 'orgUnitAttribute()'");
+    }
+    if (!orgUnitCode)  throw new Error("Missing first argument 'orgUnitCode' required by 'orgUnitAttribute()'");
+    if (!attributeKey) throw new Error("Missing second argument 'attributeKey' required by 'orgUnitAttribute()'");
```

Comment:
Ah nice touch to validate the input parameters 👍 I'd be nice if we strove towards clear and simple error messaging in the report-server to help newcomers understand how it works!